js/adapt-stateful-session.js

```diff
@@ -109,6 +110,21 @@ define([
 			}
 		},
 
+		onQuestionRecordInteraction:function(questionView, forceRecord) {
+			if (this._config._isEnabled === false) return;
+
+            if (this._config._tracking._recordInteractions === true || forceRecord) {
```

Comment:
I think the setting in config.json should be respected here - the purpose of that was to create a setting that would allow you to turn off interactions tracking completely. If you look in lines 64 - 68 of adapt-contrib-spoor.js I've already set up the code to read that in, it just needs to be uncommented.